### DISCUSSION TITLE
0.16 rich notifications?

### DISCUSSION THREAD

**Discussion by JoshuaSeidel** [blakeblackshear/frigate#15210] [Ask A Question] [answered]
are rich notifications on the table for 0.16 release?

**Comment by NickM-27** [comment]
What do you mean by rich notifications

**Reply by JoshuaSeidel** [reply]
Image thumbnails, previews. Like how they are through home assistant.

**Reply by NickM-27** [reply]
images are already supported for notifications in 0.15, provided that your browser supports it

**Reply by JoshuaSeidel** [reply]
Maybe even the option to have the genai descriptions too

**Reply by NickM-27** [reply]
that will be difficult to plumb together as notifications are for review items not specific objects.

**Reply by JoshuaSeidel** [reply]
Hmm I'm running the PWA on chrome on iOS 18 and it's just a basic notification, no thumbnail or long hold preview

**Reply by NickM-27** [reply, marked as answer]
right, because safari (which chrome on iOS is a wrapper of) does not implement image notifications currently https://developer.mozilla.org/en-US/docs/Web/API/Notification/image#browser_compatibility
